modules/claims_api/app/sidekiq/claims_api/ews_updater.rb

```diff
@@ -81,8 +49,7 @@ def update_bgs_claim(ews, bgs_claim)
                                              detail: 'Waiver update Failed', error: response[:return_code],
                                              failure_count: ews.bgs_upload_failure_count)
       else
-        ews_status = update_claim_level_suspense(ews)
-        ews.status = ews_status
+        ews.status = ClaimsApi::EvidenceWaiverSubmission::UPDATED
```

Comment:
@stiehlrod  Would it be useful to add a log out here that is has been updated?  Thinking of something like what was on lines 37 and 38 (probably do not need a variable for the message, but jsut pointing to a previous success log out).

If we do not need to see that then this can be resolved, just let me know